spec/system/task_spec.rb

```diff
@@ -1,24 +1,23 @@
 require 'rails_helper'
+include ApplicationHelper
 
 RSpec.describe 'Task', type: :system do
   describe 'Task一覧' do
+    let(:project) { create(:project) }
+    let!(:task) { create(:task, project: project) }
```

Comment:
`association`をFactoryに設定すると不要になります。
```suggestion
    let!(:task) { create(:task) }
```

Reply:
ご指摘の点について、 cf08cfe で修正しました。